leaflet-mapbox-gl.js

```diff
@@ -1,4 +1,20 @@
+function debounce (func, wait, immediate) {
+    var timeout;
+    return function () {
+        var context = this, args = arguments;
+        clearTimeout(timeout);
+        timeout = setTimeout(function () {
+            timeout = null;
+            if (!immediate) func.apply(context, args);
+        }, wait);
+        if (immediate && !timeout) func.apply(context, args);
+    };
+};
+
 L.MapboxGL = L.Layer.extend({
+    options: {
+      updateInterval: 50
```

Comment:
50 seems a little high, you would usually start with something like 16 for 60fps or 32 for 30fps...